src/main/Synthesizer.ts

```diff
@@ -362,9 +362,18 @@ export default class Synthesizer implements ISynthesizer {
 		const aLeft = new Float32Array(_module.HEAPU8.buffer, memLeft, frameCount);
 		const aRight = channels >= 2 ? new Float32Array(_module.HEAPU8.buffer, memRight, frameCount) : null;
 		if ('numberOfChannels' in outBuffer) {
-			outBuffer.copyToChannel(aLeft, 0, 0);
-			if (aRight) {
-				outBuffer.copyToChannel(aRight, 1, 0);
+			if ('copyToChannel' in outBuffer) { 
```

Comment:
This code is good, but I think you can simply write `if (outBuffer.copyToChannel)`.
By this, you can also avoid adding `any` type to `outBuffer`.
(`if (outBuffer.copyToChannel)` is not the same style to the above line `if ('numberOfChannels' in outBuffer)`, but this code is something like 'workaround' and I want to avoid using `any` as possible...)

Reply:
I updated the code as you recommended. Thanks.

Reply:
Thanks!